src/scripts/medgen_conflicts_add_xrefs.py

```diff
@@ -0,0 +1,94 @@
+"""MedGen conflicts: Add new xrefs
+
+From excel file containing workseets about MedGen conflicts, following removal of xrefs, create a robot template that
+will be used to add xrefs.
+
+See also:
+- GH issue: https://github.com/monarch-initiative/mondo-ingest/issues/273
+- Megan Kane's source files including readme.txt that goes over each sheet:
+  https://github.com/monarch-initiative/mondo-ingest/issues/273#issuecomment-1632774012
+"""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, List
+
+import pandas as pd
+
+SCRIPTS_DIR = Path(__file__).parent
+PROJECT_DIR = SCRIPTS_DIR.parent.parent
+ONTOLOGY_DIR = PROJECT_DIR / "src" / "ontology"
+TEMPLATES_DIR = PROJECT_DIR / "src" /  "templates"
+TMP_DIR = ONTOLOGY_DIR / "tmp"
+INPUT_FILE = str(TMP_DIR / "July2023_CUIReports_FromMedGentoMondo.xlsx")
+OUTPUT_FILE = str(TEMPLATES_DIR / "ROBOT_addMedGen_fromConflictResolution.tsv")
+
+
+def _rows_from_common_sheet_format(df: pd.DataFrame) -> List[Dict]:
+    """From worksheets having same common format, get prefixed xrefs for the namespaces we're looking to cover"""
+    df['xref_id'] = df['xref_id'].apply(
+        lambda x: f'MEDGENCUI:{x}' if x.startswith('CN')  # "CUI Novel"
```

Comment:
## Xref additions: From Megan's conflicts file
@matentzn I parsed through Megan's conflicts file and there are 3 sheets where she thinks xrefs should be replaced. I did this as separate steps: (i) the removals, as you have reviewed previously, and (ii) additions here as a robot template.

I've highlighted here the algorithm for prefixing each of these ids. I've used the same algorithm as the one we're using in the MedGen ingest:

https://github.com/monarch-initiative/medgen/blob/main/src/medgen2obo.pl#L161-L172

It chooses a different prefix based on if its a CN, a CUI, or a UID. And also as we did in the MedGen ingest, if it's a CUI, we add 2 xrefs: one under the `UMLS` namespace, and another under the `MEDGENCUI` namespace.

Reply:
## Next & final step: xref additions from MedGen mappings
As I wrote in [the OP](https://github.com/monarch-initiative/mondo/pull/6560#issue-1850743276), my next step is: 
> 3. Add new xrefs generated from [MedGen ingest](https://github.com/monarch-initiative/medgen): which will mainly come from the FTP file MedGenIDMappings.txt, but will probably require that whole ingest to be working to generate.

This depends on the medgen ingest working properly, and it's in pretty good shape I think, so I don't anticipate much longer until this is done too.

Reply:
## Run command to add these additions?
Is this something you'd like me to do, or would you like to do it, or will this automatically be done via some pipeline?

I'm not familiar with the exact command to do this; maybe `--merge-before` ([docs](http://robot.obolibrary.org/template#merging))?
Would this be run on `mondo-edit.obo`?